packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -47,13 +47,27 @@ function EntityRecordState( { record, checked, onChange, closePanel } ) {
 		return parents[ parents.length - 1 ];
 	}, [] );
 
+	const isSelected = useSelect(
+		( select ) => {
+			const selectedBlockId = select(
+				'core/block-editor'
+			).getSelectedBlockClientId();
+			return selectedBlockId === parentBlockId;
+		},
+		[ changesWhenSelected, parentBlockId ]
```

Comment:
maybe there is a better dep we can use for this?  I couldn't find any so I created my own boolean toggle to trigger when the button is clicked.  🤔